docs/app/pages/components/sections/panels/item-display-panel/item-display-panel.component.ts

```diff
@@ -0,0 +1,177 @@
+import { ColorService } from './../../../../../../../src/services/color/color.service';
+import { Component } from '@angular/core';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import 'chance';
+import { IPlunk } from '../../../../../interfaces/IPlunk';
+import { IPlunkProvider } from '../../../../../interfaces/IPlunkProvider';
+import { ItemDisplayPanelComponent } from '../../../../../../../src/index';
+
+@Component({
+    selector: 'uxd-item-display-panel-component',
+    templateUrl: './item-display-panel.component.html'
+})
+@DocumentationSectionComponent('ComponentsItemDisplayPanelComponent')
+export class ComponentsItemDisplayPanelComponent implements IPlunkProvider {
+
+    visible: boolean = false;
+    title: string = '';
+    top: number = 0;
+    main: string = '';
+    selected: number = 0;
+    previousEnabled: boolean = true;
+    nextEnabled: boolean = true;
+    
+    sparkBarColor: string;
+    sparkTrackColor: string;
+
+    htmlCode = require('./snippets/app.html');
+    tsCode = require('./snippets/app.ts');
+    modalDoc = require('./modalDOC.html');
+    modalPdf = require('./modalPDF.html');
+    modalPpt = require('./modalPPT.html');
+
+    items: Item[] = [{
+        id: 1,
+        name: chance.name(),
+        dateString: '3 Oct 2015',
+        document: 'Document 4.ppt',
+        extension: '.ppt',
+        storage: '95.25',
+        active: false,
+        panel: {
```

Comment:
Remove the `top` property from this as it is always the same, just put 53 in the attribute in the HTML. Also should probably rename `main` to something like `content` to make it a bit clearer about what it is